### ISSUE TITLE
version on firefox add-ons doesn't work because of division error

### ISSUE THREAD

**Issue by masukomi** [asnewman/just-read-firefox#1]
```
TypeError: SpeechSynthesisUtterance.rate setter: Value being assigned is not a finite floating-point value.
```
It's complaining about content.js line 13 which is as follows in the version downloaded from firefox add-ons

```javascript
    utterance.rate = result["utterance-speed"] / 10;
```

but in the source code here is 
```javascript
    utterance.rate = result["utterance-speed"];
```

related questions:
the plugin provides no mechanism to define the utterance speed, and if it didn't try it'd just get whatever speed was set at the OS level (presumably) which is a pretty good default.

So why is it even trying to get utterance speed? Maybe I'm missing something but I feel like it's never going to be set.

**Comment by asnewman** [comment]
Hi @masukomi, Thanks for the issue and for creating the PR. The utterance setting intends to allow users to set the speed on the fly, even with a configured OS-level speed. I often want to skim through articles with a fast setting but slow things down when digesting something more complicated.

It's definitely a problem that the extension is bugging out for you (it should still work without messing with the utterance slider I screenshot in #2 ). As a solution, I will merge your PR and follow up with more changes to move the utterance to the preference tab and make it an optional override.

**Comment by asnewman** [comment]
Just published the new version (1.2). Please let me know if you run into further issues!

**Comment by masukomi** [comment]
can confirm that it's working in the latest version.
